adagrams/game.py

```diff
@@ -1,11 +1,273 @@
+import random
+
 def draw_letters():
-    pass
+    
+    letter_pool_str= "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    # List comp for converting ABC string on line 7 into list 
+    letter_pool_list = [letter for letter in letter_pool_str]
```

Comment:
Love to see people use list comprehension! To save yourself some typing, you could have initialized this variable below `letter_pool_dict` and done:

```python
letter_pool_list = [letter for letter in letter_pool_dict.keys()]
```
Since we know we need the frequency dictionary we can leverage a view object of the keys to build this same list.